### ISSUE TITLE
Doesn't work

### ISSUE THREAD

**Issue by ThatOneCalculator** [scaryrawr/winwal#1]
```
PS C:\Users\OWNER> Update-WalTheme -Image .\Documents\desktop_ap_photo\IMG_20200107_130237.jpg
[I] image: Using image IMG_20200107_130237.jpg.
[I] theme: Set theme to IMG_20200107_130237_jpg_dark_colorthief_None_1.1.0.json.
[I] colors: Found cached colorscheme.
[I] sequences: Set terminal colors.




[I] export: Exported all files.
[I] export: Exported all user files.
True
ConvertFrom-Json : Invalid JSON primitive: .
At C:\Users\OWNER\winwal\winwal.psm1:78 char:57
+ ... nfigData = (Get-Content -Path $terminalProfile | ConvertFrom-Json)[0]
+                                                      ~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [ConvertFrom-Json], ArgumentException
    + FullyQualifiedErrorId : System.ArgumentException,Microsoft.PowerShell.Commands.ConvertFromJsonCommand

The property 'schemes' cannot be found on this object. Verify that the property exists and can be set.
At C:\Users\OWNER\winwal\winwal.psm1:87 char:5
+     $configData.schemes = $schemes
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : PropertyNotFound

Add-Member : Cannot bind argument to parameter 'InputObject' because it is null.
At C:\Users\OWNER\winwal\winwal.psm1:90 char:37
+ ... .defaults | Add-Member -MemberType NoteProperty -Name colorScheme -Va ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Add-Member], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,Microsoft.PowerShell.Commands.AddMemberCo
   mmand

Wrote selected scheme to the defaults.```

**Comment by ThatOneCalculator** [comment]
Update: It says the same error in PowerShell 7 vs the new windows terminal but it...actually works lol, but only in PowerShell 7

**Comment by scaryrawr** [comment]
> Update: It says the same error in PowerShell 7 vs the new windows terminal but it...actually works lol, but only in PowerShell 7

I'm wondering if there's an issue with 'Windows PowerShell' vs 'PowerShell Core'. I need to at least update it to stop on error instead of continuing...

I think it's handling Json differently between 5 and 7.

**Comment by scaryrawr** [comment]
Ah, it looks like PowerShell 6+ handles Json with comments correctly, Windows PowerShell won't work since the default terminal settings file has comments. There's another issue where some reason ConvertFrom-Json is returning a 2 item array where one (which doesn't appear to be consistent) is $null. I have a fix and will update shortly.

**Comment by scaryrawr** [comment]
I've updated the code a bit (still a little rough).

Your Windows Terminal settings.json may have gotten messed up, but can be regenerated from moving/deleting the file and launching a new Windows Terminal.

The new fix will create a backup copy of the settings.json that was previously there now (as settings.json.bak in the same directory as the original settings file). It also tries to handle the internal conversions a bit better.

I've also updated the README, it looks like Json handling is different in older versions of PowerShell.

Thanks for filing the issue!
